doc/blockstore/direct_data_transfer.md

```diff
@@ -0,0 +1,57 @@
+# Direct data copying between disk agents
+
+## General information
+
+Currently, a volume is used to transfer data to a new replica or shadow disk. It reads from one ghent disk and writes data to another agent disk. This leads to a doubling of the data passing through the network compared to the approach when data is transferred from one storage agent to another directly.
+
```

Comment:
"This leads to a doubling of the data passing through the network" -> "This doubles the amount of data transmitted over the network"